graph/pkg/service/v0/drives.go

```diff
@@ -360,7 +360,7 @@ func (g Graph) formatDrives(ctx context.Context, baseURL *url.URL, mds []*storag
 					// TODO cache until ./.config/ocis/space.yaml file changes
 					s = append(s, *sdi)
 				}
-				res.Special = &s
+				res.Special = s
```

Comment:
This is caused by a recent change in the code generator. I guess it's coming from this: https://github.com/OpenAPITools/openapi-generator/pull/8491  

This caused some changes in the generated go types, like here: https://github.com/owncloud/libre-graph-api-go/compare/v0.8.0...v0.9.0#diff-9379c4f5b221744a11de81c041b2961fa308fec3a32103abcc44296ee33ac283R47

:man_shrugging: 

We might wanna pin the image we're using to generate the libs at some point.

Reply:
ok
Seems this was a big improvement PR on the generator itself.
I think it brings some "good stuff".

I agree that we should pin the docker as soon as our api is supposed to be stable.